packages/stake/src/types/jest.d.ts

```diff
@@ -0,0 +1,46 @@
+import { BigNumber, Contract, Wallet } from 'ethers' // eslint-disable-line @typescript-eslint/no-unused-vars
+
+export type Numberish = number | string | BigNumber
+
+declare global {
```

Comment:
Was there a particular reason these types needed to be vendored? Not immediately clear to me :thinking:

Reply:
Types not working in the current release of the extension :D

Reply:
Got it :slightly_smiling_face: